config.yaml

```diff
@@ -1,5 +1,6 @@
 default:
  branch: 'powerkvm-v3.1.1'
+ work_dir: 'build'
```

Comment:
IMHO tmp/temp/workspace are more intuitive names than "build", but up to you guys